modules/openapi-generator/src/main/resources/rust-server/README.mustache

```diff
@@ -109,7 +109,7 @@ All URIs are relative to *{{{basePath}}}*
 
 Method | HTTP request | Description
 ------------- | ------------- | -------------
-{{#apiInfo}}{{#apis}}{{#operations}}{{#operation}}[**{{{operationIdOriginal}}}**]({{{apiDocPath}}}{{classname}}_api.md#{{{operationIdOriginal}}}) | **{{{httpMethod}}}** {{{path}}} | {{#summary}}{{{summary}}}{{/summary}}
+{{#apiInfo}}{{#apis}}{{#operations}}{{#operation}}[**{{{operationId}}}**]({{{apiDocPath}}}{{classname}}_api.md#{{{operationId}}}) | **{{{httpMethod}}}** {{{path}}} | {{#summary}}{{{summary}}}{{/summary}}
```

Comment:
I think we should keep the first one as `operationIdOriginal`, as that's what the user is going to expect to see?

Reply:
OK